authenticator.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require("jsonwebtoken")
+
+module.exports = function (req, res, next){
+
+    let authHeader = req.headers.authorization
+    let token = authHeader && authHeader.split(" ")[1]
+    //Empty token
+    if(token==null) return res.sendStatus(401)
+
+    jwt.verify(token, "SECRET", async (err, user)=>{
```

Comment:
Please make `.env` ([enviroment file](https://www.npmjs.com/package/dotenv)) to set `SECRET` key.
to make sure that we don't have to manual change a `SECRET` key later.